scormxblock/static/js/src/studio.js

```diff
@@ -117,4 +124,41 @@ function ScormStudioXBlock(runtime, element) {
     runtime.notify('cancel', {});
   });
 
+  $(element).find('input[name=display_type]').bind('click', function() {
+    showPopupLaunchType()
+  });
+
+  $(element).find('input[name=popup_launch_type]').bind('click', function() {
+    var display_type_popup = true;
+    showLaunchButtonTextField(display_type_popup)
+
+  });
+
+  function showPopupLaunchType() {
+      var display_type = $(element).find('input[name=display_type]:checked').val();
+      var display_type_popup = false;
+      if(display_type === 'popup'){
+        $(element).find('#popup_type').show();
+        display_type_popup = true;
+        showLaunchButtonTextField(display_type_popup)
+      }
+      else{
+        $(element).find('#popup_type').hide();
+        showLaunchButtonTextField(display_type_popup)
+      }
+  }
+
+  function showLaunchButtonTextField(display_type_popup) {
```

Comment:
Please change the name from `showLaunchButtonTextField` to `showHideLaunchButtonTextField` as the function shows as well as hides the options.